### PR TITLE
New Presence: Roll20

### PR DESCRIPTION
**Pull request by MrBartusek**
I have added presence for most of the [roll20.net](https://roll20.net) a popular website for playing tabletop role-playing games (like dungeons and dragons)

image
image

Supports hiding of details:
image
image
image

### CODE REVIEW

**Comment by MrBartusek** [comment]
Not sure if I should fix these `Combine this with the previous 'const' statement`?

**Comment by Bas950** [comment]
> Not sure if I should fix these `Combine this with the previous 'const' statement`?

const var1 = obj,
var2 = obj2;

File: websites/R/Roll20/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   },
+   "url": [
+     "roll20.net",
+     "app.roll20.net",
+     "marketplace.roll20.net",
+     "blog.roll20.net",
+     "wiki.roll20.net"
+   ],
+   "version": "1.0.0",
+   "logo": "https://s3.amazonaws.com/files.d20.io/marketplace/527279/nvsTfqKL38EeMpudZyiUeg/med.png?1526331193298",
```
Comment (Bas950) [inline websites/R/Roll20/dist/metadata.json:20]: upload to imgur

File: websites/R/Roll20/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ],
+   "category": "games",
+   "settings": [
+    {
+        "id": "hideDetails",
+        "title": "Hide Details",
+        "icon": "fas fa-eye",
+        "value": false
+    }
+   ]
```
Comment (Bas950) [inline websites/R/Roll20/dist/metadata.json:36]: upload a screenshot of the setting in the extension

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**